docs/HopFacet.md

```diff
@@ -4,6 +4,9 @@
 
 The Hop Facet works by forwarding Hop specific calls to the [Hop Bridge contract](https://github.com/hop-protocol/contracts/blob/master/contracts/bridges/L2_Bridge.sol). Hop works by swapping tokens into intermediate **h** tokens before bridging. For example to send **USDC** from one chain to another, we need to swap **USDC** for **hUSDC** and then send them to the Hop bridge contract. Hop provides a useful [wrapper contract](https://github.com/hop-protocol/contracts/blob/master/contracts/bridges/L2_AmmWrapper.sol#L40) to do this for us so we only need to worry about sending **USDC** and calling [`swapAndSend`](https://github.com/hop-protocol/contracts/blob/3747b0b41defddc5b3c8ca328057847cd987c2c8/contracts/bridges/L2_AmmWrapper.sol#L40-L78) on the wrapper. Native tokens like **ETH** can be sent directly to the bridge.
 
+Only _hopData.nativeFee is handled by the contract; relayerFee and bonderFee are displayed in the UI but not used on-chain.
```

Comment:
Please correct me if I am wrong but the way I understood this topic is that we are currently not using hopData.nativeFee at all.

see: https://lifi-protocol.slack.com/archives/C08CJ4FEHQA/p1742295647378249?thread_ts=1741935076.386349&cid=C08CJ4FEHQA
(which was confirmed by Max in the message after)

The comments are not accurately reflecting that imho.